test_settings.py

```diff
@@ -24,9 +24,28 @@
     },
 }
 
-INSTALLED_APPS = ("platform_plugin_aspects",)
+INSTALLED_APPS = (
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.sessions",
+    "platform_plugin_aspects",
+)
+
+ROOT_URLCONF = 'platform_plugin_aspects.urls'
+
+SECRET_KEY = 'very-insecure-key'
+
+MIDDLEWARE = [
+    'django.middleware.csrf.CsrfViewMiddleware',
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+]
```

Comment:
It's safe to remove all those settings

Reply:
I think it's [this `@login_required`](https://github.com/openedx/platform-plugin-aspects/pull/27/files#diff-1edce6254e2a9acff5c08fb9bd8d45316b5b9a287502112014844d3faa6344e2R16) that meant I had to add all this django stuff to get the tests to pass.

But once I replace it with the suggested edx-platform wrappers, I should be able to remove these.